web/src/lib/types/types.ts

```diff
@@ -353,7 +353,7 @@ export const owningFlowDisplayFields = Object.keys(emptyOwningFlowContent) as (k
 
 export const startFlowDisplayFields = Object.keys(emptyStartingFlowContent) as (keyof StartingFlowContent)[];
 
-const businessUserDisplayFields = Object.keys(emptyBusinessUser) as (keyof BusinessUser)[];
+const businessUserDisplayFields = Object.keys(createEmptyUser()) as (keyof BusinessUser)[];
```

Comment:
read below with this messing with the typing

Reply:
Is it actually causing issues? If the IDE knows the return type of the function then can't it do object.keys?

Reply:
I suppose I could override that, though... but yeah, I'm curious how this change disrupts the types?